### ISSUE TITLE
Check if destination exists

### ISSUE THREAD

**Issue by MOJNICK** [m-manu/rsync-sidekick#8]
Hi, I believe some safety checks should be added to not overwrite existing files on destination:

```
 // UnixCommand for moving or renaming a file
 func (a MoveFileAction) UnixCommand() string {
-       return fmt.Sprintf(`mv -v "%s" "%s"`, escape(a.sourcePath()), escape(a.destinationPath()))
+       return fmt.Sprintf(`mv -v -n "%s" "%s"`, escape(a.sourcePath()), escape(a.destinationPath()))
 }
 
 // Perform 'file move/rename' action
 func (a MoveFileAction) Perform() error {
-       return os.Rename(a.sourcePath(), a.destinationPath())
+    if _, err := os.Stat(a.destinationPath()); err == nil {
+        return errors.New(fmt.Sprintf(`file already exists "%s"`, a.destinationPath()))
+    } else if errors.Is(err, os.ErrNotExist) {
+           return os.Rename(a.sourcePath(), a.destinationPath())
+    } else {
+        return err
+    }
 }
```

**Comment by m-manu** [comment]
@MOJNICK Can you please send a pull request?

**Comment by MOJNICK** [comment]
Sure, I need more privileges:

> git push --set-upstream origin make_moving_safer 
> ERROR: Permission to m-manu/rsync-sidekick.git denied to MOJNICK.
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.

**Comment by m-manu** [comment]
@MOJNICK Please follow below steps: 

1. [Fork this repository](https://github.com/m-manu/rsync-sidekick/fork) into your username
2. Make changes 
3. Send pull request

**Comment by m-manu** [comment]
@MOJNICK I've merged your diff to v1.4.0
